content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -206,10 +206,10 @@ rules:
 [Cloud Controller Manager Administration](/docs/tasks/administer-cluster/running-cloud-controller/#cloud-controller-manager)
 has instructions on running and managing the cloud controller manager.
 
-Want to know how to implement your own cloud controller manager, or extend an existing project?
+Want to know how to implement your own cloud controller manager or extend an existing project?
```

Comment:
The original seems easier to parse, if the intended meaning is:

> Want to:
> - know how to implement your own cloud controller manager?
> - extend an existing project?

If rewording we could mention that the advice applies to CCM implementations in Go. Implementations in other languages are possible if you like hard work :wink: